src/jobs/retrigger.js

```diff
@@ -14,8 +14,11 @@ import Joi from 'joi';
 // We use public only operations on the queue here...
 const queue = new taskcluster.Queue();
 
-function recursiveUpdateTaskIds(tasks, map) {
-  return traverse.map(tasks, (value) => {
+export function recursiveUpdateTaskIds(tasks, map) {
+  return traverse.map(tasks, function (value) {
+    // Do not attempt to change old task Ids
+    if (this.key === 'oldTaskId') return value;
```

Comment:
This ensures that we do not overwrite the oldTaskId with the new task id making later verification and logging incorrect.